src/pymovements/dataset/dataset_download.py

```diff
@@ -31,6 +32,10 @@
 from pymovements.dataset.dataset_paths import DatasetPaths
 
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('__name__')
```

Comment:
the string `'__name__'` should be replaced with the dunder variable `__name__`, otherwise the info source is not the module but just `__name__`.

(`__name__` will resolve to `pymovements.dataset.dataset_download`)

Reply:
good catch!